bedrock/firefox/templates/firefox/campaign/unfck/index.html

```diff
@@ -21,25 +21,22 @@
 
 {% macro item(title=None, info=None, id=None, src=None, src_still=None, include_cta=False,
               link_text=None, link_url=None, link_include_utms=True, link_is_external=True) -%}
-  <div class="c-item-unfck" id="{{ id }}">
-    <h3 class="c-item-title">{{ title }}</h3>
-    <div class="c-item-img">
-      <picture class="c-item-gif">
-        {% if src_still %}
-        <source srcset="{{ src_still }}" media="(prefers-reduced-motion: reduce)" loading="lazy" width="280" height="280"></source>
-        {% endif %}
-        <img src="{{ src }}" loading="lazy" width="344" height="344">
-      </picture>
-    </div>
-    <div class="c-item-desc">
-      <p>{{ info }}</p>
-      {% if include_cta %}
-        {{ caller() }}
-      {% else %}
-        <p><a class="mzp-c-cta-link" href="{{ link_url }}{% if link_include_utms %}{{ referrals }}{% endif %}" target="_blank" {% if link_include_utms or link_is_external %}rel="external noopener" data-cta-type="link" data-cta-text="Checklist: {{ id }}"{% endif %}>{{ link_text }}</a></p>
-      {% endif %}
-    </div>
-  </div>
+    <a id="{{ id }}" class="c-item-unfck" href="{{ link_url }}{% if link_include_utms %}{{ referrals }}{% endif %}" target="_blank" {% if link_include_utms or link_is_external %}rel="external noopener" data-cta-type="link" data-cta-text="Checklist: {{ id }}"{% endif %}>
+      <h3 class="c-item-title">{{ title }}</h3>
+      <div class="c-item-img">
+        <picture class="c-item-gif">
+          {% if src_still %}
+          <source srcset="{{ src_still }}" media="(prefers-reduced-motion: reduce)" loading="lazy" width="280" height="280"></source>
+          {% endif %}
+          <img src="{{ src }}" loading="lazy" width="344" height="344">
```

Comment:
Should the image sizes here match `<source>`?
```suggestion
          <img src="{{ src }}" loading="lazy" width="280" height="280">
```